polaris/polaris/management/commands/watch_transactions.py

```diff
@@ -140,9 +206,15 @@ def find_matching_payment_op(
         except KeyError:
             return
 
-        # memo from response must match transaction.memo
+        # memo from response must match transaction memo
         memo = response.get("memo")
-        if memo != transaction.withdraw_memo:
+        if (
```

Comment:
This should mean that its sep6 right?  Why not `transaction.protocol == Transaction.PROTOCOL.sep6`?

Reply:
Nope. This condition is to make sure we're comparing `memo` with the right column on `transaction`. We could do `transaction.protocol in [Transaction.PROTOCOL.sep6, Transaction.PROTOCOL.sep24]` OR `transaction.protocol != Transaction.PROTOCOL.sep31`